apps/studio/prisma/seed.ts

```diff
@@ -86,13 +86,8 @@ async function main() {
   await db
     .insertInto('Navbar')
     .values({
-      id: '1',
-      siteId: '1',
-      content: {
-        name: 'navi',
-        url: 'www.isomer.gov.sg',
-        items: NAV_BAR_ITEMS,
-      } satisfies Navbar,
+      siteId: id,
+      content: { items: NAV_BAR_ITEMS } satisfies Navbar,
```

Comment:
question(non-blocking): are we able to type `NAV_BAR_ITEMS` to `Navbar` directly when we define it above in line 14?

Reply:
added on otp, still kept check here